### PR TITLE
a

### PR DESCRIPTION
**Pull request by Devan-Kerman**

### CODE REVIEW

**Comment by Devan-Kerman** [comment]
> incompatible with my pr plsfix

no fuq u >:(

**Comment by valoeghese** [comment]
did you use proguard on the pr name

**Comment by valoeghese** [comment]
Conflicts

File: src/main/resources/everything.mixins.json [hunk 1]
```diff
@@ -5,26 +5,28 @@
...
     "a319fec2123443439bafdfe06d9267d7",
+    "CopperOreFix",
     "Event",
-    "MixinVanillaLayeredBiomeSource",
-    "MixinSetBaseBiomesLayer",
-    "MixinContinentLayer",
     "implementation.event.BlockEventsImpl",
     "implementation.event.ServerMainImpl",
     "implementation.event.ServerPlayNetworkHandlerImpl",
-    "implementation.event.PlayerManagerImpl"
```
Comment (valoeghese) [inline src/main/resources/everything.mixins.json:15]: why is this moved from mixins to client
